x-pack/plugins/infra/server/services/inventory_views/inventory_views_client.ts

```diff
@@ -186,6 +186,7 @@ export class InventoryViewsClient implements IInventoryViewsClient {
     id: staticInventoryViewId,
     attributes: {
       ...staticInventoryViewAttributes,
+      time: Date.now(),
```

Comment:
since staticInventoryViewAttributes already says Date.now() how come we have to do it again here?

Reply:
Because `staticInventoryViewAttributes` is an exported object and the static view is generated on the build when the server compiles the file, that's why it keeps the same date each time and is not regenerated, as it should be